Recruit-Training/Advanced-Recruit-Training/Telemetry-Teaser/src/app/app.component.html

```diff
@@ -1,18 +1,36 @@
 <body>
   <div>
     <h1> Elysia Range Simulator </h1>
-    <form name="simulator" #simulator="ngForm" novalidate>
+    <form [formGroup]="rangeForm" name="simulator" #simulator="ngForm" novalidate>
       <label>Speed (km/h):</label>
       <br>
-      <input name="speed" type="number" placeholder="Speed" (keyup)="onKeySpeed($event)">
+      <input class = "form-control" formControlName = "speedInput" name="speed" type="number" placeholder="Speed" (keyup)="onKeySpeed($event)">
+      <br>
+      <div *ngIf="speedInput.invalid">
+        <label *ngIf="speedInput.errors.required" [ngClass] = "'error'">Speed is required</label>
```

Comment:
It is convention to have no spaces between operators in our html files. So [ngClass] = "'error'" can be [ngClass]="'error'"
Please implement this change to all similar cases in the html file